Makefile

```diff
@@ -81,7 +81,7 @@ test-with-cover:
 
 .PHONY: addlicense
 addlicense:
-	$(ADDLICENCESE) -y '' -c 'OpenTelemetry Authors' $(ALL_SRC)
+	$(ADDLICENCESE) -c 'The OpenTelemetry Authors' $(ALL_SRC)
```

Comment:
Do we need "The"?

Reply:
Just trying to be consistent with the other repos but I'd be happy to drop it.